### PR TITLE
Fixes the gas filters.

### PR DESCRIPTION
**Pull request by PJB3005**
Neither me nor @DrCelt knows how the fuck the last PR broke filters so badly, so I reverted it partially and moved the `..()`, which fixes it (albeit shittily).

### CODE REVIEW

File: code/ATMOSPHERICS/components/trinary_devices/filter.dm [hunk 1]
```diff
@@ -1,8 +1,7 @@
 obj/machinery/atmospherics/trinary/filter
 	icon = 'icons/obj/atmospherics/filter.dmi'
 	icon_state = "hintact_off"
-	var/base_state = "hintact_"
-	name = "\improper Gas filter"
+	name = "Gas filter"
```
Comment (dylanstrategie) [inline code/ATMOSPHERICS/components/trinary_devices/filter.dm]: Shouldn't be capitalized

**Comment by N3X15** [comment]
Has this been tested

**Comment by N3X15** [comment]
Fuck it

**Comment by PJB3005** [comment]
@N3X15 it has, if you're still wondering, I was asleep.

File: code/ATMOSPHERICS/components/trinary_devices/filter.dm (hunk 1 above)
Comment (PJB3005) [inline code/ATMOSPHERICS/components/trinary_devices/filter.dm]: Ah fuck I forgot to not revert that, damnit.

**Pull request merged**